help/destinations/catalog/data-partner/acxiom-audience-connectors.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Acxiom Audience Connectors
+description: Provides access to build an audience--hydrated by [!DNL Acxiom's Real ID] technology for improved match rates and precise scale--and send that audience to selected destinations.
+last-substantial-update: 2024-06-17
+badge: Beta
+exl-id: 
+---
+# [!DNL Acxiom Audience Connectors] destination connection
+
+>[!NOTE]
+>
+>The [!DNL Acxiom Audience Connectors] destination connection is in beta. This destination connector and documentation page are created and maintained by the [!DNL Acxiom] team. 
+
+
+Use the [!DNL Acxiom Audience Connectors] connector to build an audience--hydrated by [!DNL Acxiom's Real ID] technology for improved match rates and precise scale--and send that audience to selected destinations.
+
+This tutorial provides instructions to create an [!DNL Acxiom Audience Connectors] destination connector using the [!DNL Adobe Experience Platform] user interface. This connector is used to build and distribute audiences to selected destinations.
+
+## Use cases {#use-cases}
+
+To help you better understand how and when you should use the [!DNL Acxiom Audience Connectors] destination, here is a sample use case that [!DNL Adobe Experience Platform] customers can solve by using this connector.
+
+### Send audiences from an Experience Platform application to your Acxiom account {#send-audiences}
+
+This connector should be used by marketing professionals aiming to send audiences from an [!DNL Experience Platform] application to your [!DNL Acxiom] account for cross-channel acquisition.

```

Comment:
Per our tech editor:

"
I changed your suggestion slightly to this: 
Use this destination connector if you are a marketing professional who wants to send audiences from [!DNL Experience Platform] to your [!DNL Acxiom] account, for cross-channel acquisition.

Because we are speaking to the reader when we say “If you are a marketing professional” then we need to continue with “your [!DNL Acxiom] account” instead of “their” as suggested. 
"